src/tests/system/tests/test_ldap.py

```diff
@@ -16,20 +16,37 @@
 
 @pytest.mark.ticket(bz=[795044, 1695574])
 @pytest.mark.importance("critical")
-@pytest.mark.authentication
 @pytest.mark.parametrize("modify_mode", ["exop", "ldap_modify"])
-@pytest.mark.topology(KnownTopology.LDAP)
-def test_ldap__password_change(client: Client, ldap: LDAP, modify_mode: str):
+@pytest.mark.parametrize("use_ppolicy", ["true", "false"])
+@pytest.mark.parametrize("sssd_service_user", ("root", "sssd"))
+@pytest.mark.skipif(
+    bool(lambda client, sssd_service_user: ((sssd_service_user != "root") and not client.features["non-privileged"])),
```

Comment:
No, don't pull this into sssd-2.9

Reply:
Removed the test entirely. The normal policy tests will covered under generic tests.